hoomd/mpcd/update.py

```diff
@@ -29,28 +29,39 @@ class ReverseNonequilibriumShearFlow(Updater):
 
         slab_width (float): Width of momentum-exchange slabs.
 
-        target_momentum (float): Target momentum for swapped particles.
+        target_momentum (float): Target momentum for swapped particles. This
+            argument has a default value of infinity but can be
+            redefined with positive real numbers only.
 
-    This updater generates a bidirectional shear flow in *x* by imposing a momentum
-    flux on the system in *y*.
+    This updater generates a bidirectional shear flow in *x* by imposing a
+    momentum flux on the system in *y*.
     There are two exchange slabs separated by a distance of :math:`L_y/2`. The
     edges of these slabs are located at (:math:`-L_y/2`, :math:`-L_y/2` +
     `slab_width`) and (:math:`0.0`, `slab_width`) along the *y*-direction
-    (gradient direction) of the simulation box. On each `trigger`, particles are
-    sorted into the exchange slabs based on their positions in the box. In each
-    slab, particles whose *x*-component momenta are near `target_momentum` are
-    selected for a pairwise momentum swap, and up to `num_swaps` swaps are
-    executed. Candidate particles for swapping are those with positive
-    *x*-component momentum (flow direction) in the lower slab and those with
-    negative *x*-component momentum in the upper slab.
+    (gradient direction) of the simulation box. On each `trigger`, particles
+    are sorted into the exchange slabs based on their positions in the box.
+    Particles whose *x*-component momenta are near `target_momentum`
+    in the lower slab, and those near -`target_momentum` in the upper
+    slab, are selected for a pairwise momentum swap. Up to `num_swaps` swaps
+    are executed per update.
+
+    The amount of momentum transferred between the exchange slabs is known.
+    Therefore `summed_exchanged_momentum`, which returns the accumulated
+    momentum exchanged till the current timestep, is used to calculate the
+    momentum flux, :math:`j_{xy}`. The shear rate, :math:`\dot{\gamma}`, is
+    also extracted as the gradient of the linear velocity profile developed
+    from the flow. The viscosity can then be computed from these two quantities
+    as:
+
+    .. math::
+
+        \eta (\dot{\gamma}) = \frac{j_{xy}}{\dot{\gamma}}.
```

Comment:
@mphoward, typically the viscosity expression comes with the minus sign. In our case, our calculated flux is positive so I have ignored the minus sign in the above relation. What are your thoughts on this?

Reply:
You should write the equation in the way that makes sense for use with our code. I think we determined that the minus sign is effectively absorbed into the momentum exchanged based on the direction of the swap. You can make clear which direction the momentum exchange is defined.

Reply:
I think the equation in this way follows with our code usage. I will clarify the direction of momentum exchange then.